web/package.json

```diff
@@ -60,8 +62,8 @@
     "lint": "eslint './src/**/*.{ts,tsx}'",
     "lint:fix": "eslint --fix './src/**/*.{ts,tsx}'",
     "format": "prettier --write \"**/*.+(js|jsx|json|yml|yaml|css|md)\"",
-    "storybook": "start-storybook -p 6006 -s public",
-    "build-storybook": "build-storybook -s public"
+    "storybook": "export $(cat .env | xargs) && start-storybook -p 6006 -s public",
```

Comment:
Which env variables does storybook need?

Reply:
Probably none. There was one but then I reverted the changes. I kinda want to leave it this way tho just to be precautious.